content/applications/productivity/iot/devices/camera.rst

```diff
@@ -2,44 +2,40 @@
 Connect a Camera
 ================
 
-When using your *IoT Box* in Odoo, you could need to use a camera. As
-it can be done in a few steps, this device is really easy to configure.
-Then, you can use it in your manufacturing process and link it to a
-control point. Doing so will allow you to take pictures when you reach
-the chosen quality control point.
+A camera can be used when using the *IoT Box* with an Odoo database. It can be added in a
+few easy steps. Then, it can be used in a manufacturing process and it can be linked to a control
+point. Doing so will allow for the taking of pictures when a chosen quality control point has been
+reached.
 
 Connection
 ==========
 
-To connect the camera to the *IoT Box*, simply connect the two by
-cable.
+To connect the camera to the *IoT Box*, simply connect the two by cable. This is likely a USB cable.
 
-If your camera is a `*supported
-one* <https://www.odoo.com/page/iot-hardware>`__, there is no need to
+If the camera is a `supported one <https://www.odoo.com/page/iot-hardware>`__, there is no need to
 set up anything as it will be detected as soon as it is connected.
 
 .. image:: camera/camera_01.png
    :align: center
+   :alt: Camera recognized on the IoT box.
 
-Link a Camera to a Quality Control Point
-========================================
+Link a Camera to a quality control point within a manufacturing process
+=======================================================================
 
-With the Manufacturing app
---------------------------
+In the :menuselection:`Quality app`, a device can be setup on a *Quality Control Point*. Go to the
+:menuselection:`Control Points --> Quality Control` and open the control point which will be linked
+with the camera.
 
-In your *Quality app*, you can setup the device on a *Quality Control
-Point*. Go to the *Control Points* menu, under *Quality Control*
-and open the control point you want to link with the camera.
-
-Now, you can edit the control point and select the device from the
-dropdown list. Now, hit save.
+Now, edit the control point and choose the device from the dropdown list. The changes will
+automatically save when navigating away from the page with the changes.
```

Comment:
can be misleading as this behavior was introduced in version >= 15. This PR is targeting 14 so it's technically wrong.
Maybe change it to something like "make sure it is saved".

Reply:
Great Catch, thank you. I've made this change.